vspace/src/main/webapp/WEB-INF/views/staff/images/imagelist.html

```diff
@@ -1,5 +1,7 @@
 <html layout:decorate="~{layouts/main_staff}">
 <head>
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
```

Comment:
Let's remove it from here and put it in the layout template and then also remove it from file.html and slide.html.